src/components/TodoMain/TodoMain.tsx

```diff
@@ -0,0 +1,100 @@
+import cn from 'classnames';
+import { useState } from 'react';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[] | null;
+  tempTodo: Todo | null;
+  deleteTodo: (id: number) => void;
+};
+
+export const TodoMain: React.FC<Props> = ({ todos, tempTodo, deleteTodo }) => {
+  const [isEditActive, setIsEditActive] = useState<number>(-1);
```

Comment:
why is it number? What exactly are you checking here?

Reply:
When user double clicks on a todo, I'll save the id in here to activate Edit for it. I'll change the name...

Reply:
still not clear and looks kinda hardcoded. There must be a better way. Could you pls explain me the issue in more details?

Reply:
Here is the thing:
I have 10 todos with id [1, 2, 3, ..., 10]
They're rendered on the page with key={1, 2, 3, ..., 10}
When I want to activate Edit form on one of them, I'll pass the key/id of that todo in order to select it...

I've tried passing a boolean and end up activating Edit form for all of the todos....